test/Cnab.Tests/BundleTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Cnab.Bundle;
+
+namespace Cnab.Tests
+{
+    [TestClass()]
+    public class BundleTests
+    {
+        [TestMethod]
+        public void TestLoadUnsigned()
+        {
+            var bundle = Bundle.Bundle.LoadUnsigned("../../../../examples/bundles/thin-bundle.json");
+
+            Assert.AreEqual("helloworld", bundle.Name);
+            Assert.AreEqual("0.1.2", bundle.Version);
+            Assert.AreEqual("An example 'thin' helloworld Cloud-Native Application Bundle", bundle.Description);
+
+            Assert.AreEqual("Matt Butcher", bundle.Maintainers[0].Name);
+            Assert.AreEqual("technosophos@gmail.com", bundle.Maintainers[0].Email);
+            Assert.AreEqual("https://example.com", bundle.Maintainers[0].Url);
+
+            Assert.AreEqual("docker", bundle.InvocationImages[0].ImageType);
+            Assert.AreEqual("technosophos/helloworld:0.1.0", bundle.InvocationImages[0].Image);
+            Assert.AreEqual("sha256:aaaaaaa...", bundle.InvocationImages[0].Digest);
+
+            Assert.AreEqual("technosophos/microservice:1.2.3", bundle.Images["my-microservice"].Image);
+            Assert.AreEqual("my microservice", bundle.Images["my-microservice"].Description);
+            Assert.AreEqual("sha256:aaaaaaaaaaaa...", bundle.Images["my-microservice"].Digest);
+            // TODO - see https://github.com/deislabs/cnab-spec/issues/89
```

Comment:
Currently, the `uri` field in `image` is only present in some examples, missing from the spec and JSON schema. See https://github.com/deislabs/cnab-spec/issues/89 for more context.